gooddata-java/src/main/java/com/gooddata/sdk/service/connector/ConnectorService.java

```diff
@@ -260,6 +261,45 @@ public void updateSettings(final Project project, final Settings settings) {
         return createProcessPollResult(process.getUri());
     }
 
+    /**
+     * Get Zendesk reload.
+     *
+     * You should use the result of {@link #scheduleZendesk4Reload} to see changes in {@link Reload#getStatus()} and
+     * {@link Reload#getProcessId()} or retrieve process URI {@link Reload#getProcessUri()}.
+     *
+     * @param reload existing reload.
+     * @return same reload with refreshed properties (status, processId, process URI)
+     */
+    public Reload getZendesk4Reload(final Reload reload) {
+        notNull(reload, "reload");
+        isTrue(reload.getUri().isPresent(), "reload.uri");
+        return getZendesk4Reload(reload.getUri().get());
+    }
+
+    /**
+     * Get Zendesk reload.
+     * @param reloadUri existing reload URI
+     * @return reload
+     */
+    public Reload getZendesk4Reload(final String reloadUri) {
```

Comment:
Please rename to `getZendesk4ReloadByUri`

Reply:
Done